docker-compose.yml

```diff
@@ -19,15 +21,86 @@ services:
       - AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
       - AWS_SECRET_KEY=${AWS_SECRET_KEY}
       - ENABLE_BACKUP_MANAGEMENT=1
+    #      - PERCONA_TEST_PLATFORM_ADDRESS=https://check.localhost
+    #      - PERCONA_TEST_PLATFORM_INSECURE=1
+    #      - PERCONA_TEST_PLATFORM_PUBLIC_KEY=<public key>
+    #      - PERCONA_TEST_TELEMETRY_INTERVAL=10s
+    #      - PERCONA_TEST_TELEMETRY_RETRY_BACKOFF=10s
+    #      - PERCONA_TEST_PMM_CLICKHOUSE_ADDR=127.0.0.1:9000
+    #      - PERCONA_TEST_PMM_CLICKHOUSE_DATABASE=pmm
+    #      - PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE=10000
+    #      - PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE=2
+    #      - PMM_DEBUG=1
+
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    #      - portal.localhost:${PORTAL_HOST:-host-gateway}
+    #      - check.localhost:${PORTAL_CHECK_HOST:-host-gateway}
+    #      - pmm.localhost:${PORTAL_PMM_HOST:-host-gateway}
+    #      - check-dev.percona.com:${PORTAL_PMM_HOST:-host-gateway}
+
+    # for delve
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - seccomp:unconfined
+
+    # see https://github.com/golang/go/wiki/LinuxKernelSignalVectorBug#what-to-do
+    ulimits:
+      memlock: 67108864
+
+    ports:
+      - ${PMM_PORT_HTTP:-80}:80
+      - ${PMM_PORT_HTTPS:-443}:443
+      # For headless delve
+      - ${PMM_PORT_DELVE:-2345}:2345
+      # PG
+      - ${PMM_PORT_PG:-15432}:5432
+      # CH
+      - ${PMM_PORT_CH_TCP:-11000}:9000
+      - ${PMM_PORT_CH_HTTP:-11123}:8123
+    volumes:
+      - ./:/root/go/src/github.com/percona/pmm
+      - ./Makefile.devcontainer:/root/go/src/github.com/percona/pmm/Makefile:ro # change Makefile in devcontainer
+      # caching
+      - go-modules:/root/go/pkg/mod
+      - root-cache:/root/.cache
+
+  # PMM with external ClickHouse DB
+  ch:
+    profiles:
+      - pmm-ch
+    image: ${CH_IMAGE:-clickhouse/clickhouse-server:22.6.9.11-alpine}
+    platform: linux/amd64
+    hostname: ${CH_HOSTNAME:-ch}
+    ports:
+      - ${CH_PORT:-9000}:9000
+  pmm-managed-server-ch:
```

Comment:
`ch` and `pmm-managed-server-ch` will be used when develop runs container the following way:

```
PROFILES=pmm-ch make env-up
```